src/tooltip/css/objects/_honeycomb.tooltip.objects.tooltip.scss

```diff
@@ -0,0 +1,96 @@
+
+/* Base styles for the entire tooltip and default position*/
+.tooltip, [data-tooltip], .tooltip-top {
+  position: relative;
+  cursor: help;
+
+  /* Shared animations and properties */
+  &:before, &:after {
+    position: absolute;
+    visibility: hidden;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0);
+    opacity: 0;
+    -webkit-transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            -webkit-transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    -moz-transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            -moz-transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    -webkit-transform: translate3d(0, 0, 0);
+    -moz-transform:    translate3d(0, 0, 0);
+    transform:         translate3d(0, 0, 0);
```

Comment:
Is adding -moz- prefixes something that you guys would want? I'm more than happy to look in to this myself in 10% time tomorrow. I was planning to continue working on this then anyway.